app/build.gradle

```diff
@@ -21,13 +21,13 @@ android {
     buildToolsVersion "25.0.2"
 
     defaultConfig {
-        applicationId "io.intrepid.contest"
+        applicationId "io.intrepid.coverart"
```

Comment:
Careful, this got changed again. Be sure to self review your PRs before opening them to catch small things like this, it helps speed up the process so you can merge faster.